pkg/module/filter.go

```diff
@@ -118,6 +119,7 @@ func (f *Filter) initFromConfig() {
 	lines, err := getConfigLines()
 
 	if err != nil || len(lines) == 0 {
+		f.Off = true
```

Comment:
can we also have a ENV override? what do you think? even with a filter file provided, we could turn it off

Reply:
@michalpristas I like that. I'll add one. I think the Filter should be Off by default unless provided with an actual configuration (or ENV override). But once we have Olympus working well, then maybe the Filter should be On by default unless the user explicitly turns it off by ENV only.

Reply:
Could we also skip the whole middleware if the environment var is set?

Reply:
@michalpristas i liked your ENV idea more than my "Off" idea that I basically removed my code 😄 -- it's much easier to just not use the middleware at all if we want it off.